### PR TITLE
Adds report

### LINKED ISSUE #62: Fix the report
We might need to write a report regarding our results. They're currently all over the place (mostly in the code as comments). 
We do have a template we could fill in here:
https://github.com/SEFGroup23/assignment3-coverage/blob/main/resources/report.md

### PR DESCRIPTION
**Pull request by capitanu**
Fixes #62

### CODE REVIEW

File: resources/report.md [hunk 1]
```diff
@@ -6,94 +6,127 @@ Name: Pythonic Data Structures and Algorithms
...
    * Did all methods (tools vs. manual count) get the same result?
+   
+	 We got a pretty evenly balanced result of similar and different outputs 
+	 from the tool and our manual. All the outputs are located in each of the files.
+	 
    * Are the results clear?
+   
+	 The results seem to be pretty clear for our manual computations. After further
+	 inverstigations, we figured out that the tool does not take the final return
+	 statements into account when compute the CC.
```
[the remark this answers is not in the document]
Comment (ArnPellesGit) [reply resources/report.md:44]: sure will do in a bit

File: resources/report.md [hunk 2]
```diff
@@ -6,61 +6,61 @@ Name: Pythonic Data Structures and Algorithms
 
 URL: https://github.com/SEFGroup23/algorithms-1
 
-This project is a compilation of multiple classic algorithms in 
+This project is a compilation of multiple classic algorithms in
```
Comment (ArnPellesGit) [inline resources/report.md:9]: I did not add/remove these stupid spaces. freakin atom. Ignore them

**Comment by capitanu** [comment]
Looks good to me! I can't approve it tho since I am the author

**Comment by ArnPellesGit** [comment]
> Looks good to me! I can't approve it tho since I am the author

Am I allowed to approve? I changed it too

**Review by ArnPellesGit — APPROVED** [review]
Lgtm

**Pull request merged**